demisto_sdk/commands/common/errors.py

```diff
@@ -2162,7 +2162,8 @@ def runas_is_dbotrole():
     def changed_pack_name(original_name):
         return f'Pack folder names cannot be changed, please rename it back to {original_name}.' \
                f' If you wish to rename the pack, you can edit the name field in pack_metadata.json,' \
-               f' and the pack will be shown in the Marketplace accordingly.'
+               f' and the pack will be shown in the Marketplace accordingly.\n' \
+               f'If the file wasnt renamed, try to pull changes from master and re-run validations'
```

Comment:
what does `if the file wasnt renamed` mean? Does it not show properly on the marketplace?
if so: 

```suggestion
               f' and the pack will be shown in the Marketplace accordingly.\n' \
               f'If the new name does not show in the marketplace properly, try merging from content/master and re-running validations.'
```

Reply:
In case you're not merged from master the validation that throws this exception might mistake files for being renamed even if they weren't